i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-wsus-nsclient.md

```diff
@@ -1,129 +1,273 @@
 ---
 id: applications-wsus-nsclient
-title: Microsoft WSUS NSClient++
+title: Microsoft WSUS Server
```

Comment:
The exact name of the Pack is "Microsoft WSUS", shouldn't it be the same for the doc ? (My preference goes for the "Server" suffix but this implies to adjust the pack...)